src/components/Header/Siderbar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import Texts from '../Texts'
+import { TEXTS_SIDEBAR } from '../Header.data'
+import CloseIcon from '@mui/icons-material/Close'
+
+type SideBarProps = {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const SideBar = ({ isOpen, onClose }: SideBarProps) => {
+  return (
+    <nav
+      className={`
+          right-0
+          top-0
+          h-screen
+          bg-sky-100
+          flex
+          flex-col
+          gap-2
+          justify-start
+          items-center
+          p-7
+          max-w-xs
+          ${isOpen ? 'absolute' : 'hidden'}
+        `}
+    >
+      <CloseIcon className='self-end w-7' onClick={() => onClose()} />
```

Comment:
Nesse caso você pode passar a referência da função, sem precisar criar uma função anônima =)


```suggestion
      <CloseIcon className='self-end w-7' onClick={onClose} />
```

Reply:
Bela sugestão!

Feito: https://github.com/gabrielduete/virtual-healthcare/pull/2/commits/ca354afc658b24948d41a81c74bee4a3e9e0292a